README.md

```diff
@@ -0,0 +1,196 @@
+# Hazelcast Discovery Plugin for Kubernetes
+
+This repository contains a plugin which provides the automatic Hazelcast member discovery in the Kubernetes environment.
+
+You can use it in your project deployed on Kubernetes in order to make the embedded Hazelcast members discover each other automatically. This plugin is also included in Hazelcast Docker images, Hazelcast Helm Charts, and Hazelcast OpenShift Docker image.
+
+## Embedded mode
```

Comment:
I suggest to mention hazelcast java client connecting to an existing hazelcast cluster in k8s. That is another valid use case. We might include a sample config for that too.

Reply:
Added in the section at the bottom. PTAL.